packages/contracts/contracts/ux/CreateNewsroomInGroup.sol

```diff
@@ -0,0 +1,27 @@
+pragma solidity ^0.4.23;
+
+import "../newsroom/NewsroomFactory.sol";
+import "../proof-of-use/groups/UserGroups.sol";
+
+contract CreateNewsroomInGroup {
+  NewsroomFactory factory;
+  UserGroups groups;
+
+  constructor(NewsroomFactory _factory, UserGroups _groups) {
+    factory = _factory;
+    groups = _groups;
+  }
+
+  function create(string name, string charterUri, bytes32 charterHash, address[] initialOwners, uint initialRequired)
+    public
+    returns (Newsroom newsroom)
+  {
+    require(initialOwners.length > 0);
+
+    newsroom = factory.create(name, charterUri, charterHash, initialOwners, initialRequired);
+
+    groups.allowInGroupTransfersAll(initialOwners);
```

Comment:
I don't intend to ever pass in an array of initial owners.  I don't see a reason to do that.  Its just the generated contracts that need to be added.